python-sdk/src/astro/sql/operators/run_query.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Callable, Optional
+
+from airflow.decorators.base import get_unique_task_id
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+
+from astro.databases import create_database
+
+
+class RunQueryOperator(BaseOperator):
+    """
+    Run a SQL statement and return the result of execution
+
+    :param sql_statement: sql query to execute
+    :param conn_id: Airflow connection id
```

Comment:
Can you expand what this connection would be used for?

Reply:
Improved this param description please let me know it looks now